pkg/provisioner/crdprovisioner.go

```diff
@@ -177,12 +178,13 @@ func (a CachedReader) List(_ context.Context, objectList client.ObjectList, opts
 	return err
 }
 
-func NewCrdProvisioner(azdiskClient azdisk.Interface, objNamespace string, cw *watcher.ConditionWatcher, azCachedReader CachedReader, informer azureutils.GenericInformer) *CrdProvisioner {
+func NewCrdProvisioner(azdiskClient azdisk.Interface, objNamespace string, cw *watcher.ConditionWatcher, azCachedReader CachedReader, informer azureutils.GenericInformer, waitForLunEnabled bool) *CrdProvisioner {
```

Comment:
Instead of adding a new parameter, let's replace the `objNamespace string` parameter with a `config *azdiskv1beta2.AzDiskDriverConfiguration` parameter.

Reply:
fixed